health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/CreateChecklist.js

```
@@ -636,14 +517,12 @@ const CreateChecklist = () => {
                   value={t(pair.value)} // Dynamically set the value
                 // style={{ fontSize: "16px", fontWeight: "bold" }} // Optional: customize styles
                 />
-                <div style={{ height: "1rem" }}></div>
+                {(index !== (fieldPairs.length - 1)) && <div style={{ height: "1rem" }}></div>}
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)